airflow/www/views.py

```diff
@@ -108,8 +108,13 @@ def get_safe_url(url):
     valid_schemes = ['http', 'https', '']
     valid_netlocs = [request.host, '']
 
+    # Remove single quotes
+    url = url.replace("'", "")
     parsed = urlparse(url)
 
+    query = parse_qsl(parsed.query)
```

Comment:
```suggestion
    query = parse_qsl(parsed.query, keep_blank_values=True)
```

Reply:
fixed in https://github.com/apache/airflow/pull/12459/commits/7486153f451e4d2bb1c6fd9cbb5a63430157c99c